### PR TITLE
Mrview fix command line parsing

### PR DESCRIPTION
**Pull request by maxpietsch**
fixes segfault and assert failure in command line parsing as described in #1006

### CODE REVIEW

File: src/gui/mrview/window.cpp [hunk 1]
```diff
@@ -2012,7 +2013,7 @@ namespace MR
             return;
           }
 
-          assert ("shouldn't reach here!" && false);
+          assert (opt.opt->is ("info") or opt.opt->is ("debug") or ("shouldn't reach here!" && false));
```
Comment (jdtournier) [inline src/gui/mrview/window.cpp]: Struggling a little to understand how this even compiles...? What are these 'or' statements doing in there? This is supposed to be C++, not python... 😕

Comment (Lestropie) [inline src/gui/mrview/window.cpp]: http://en.cppreference.com/w/cpp/keyword/or

Is [C++98](http://www.bezem.de/pdf/ReservedWordsInC.pdf) onward apparently.

Comment (maxpietsch) [inline src/gui/mrview/window.cpp]: [seems to work](https://qph.ec.quoracdn.net/main-qimg-d2762148238117d097dba8367d42bf33-c)

Comment (jdtournier) [inline src/gui/mrview/window.cpp]: Wow! I never knew that... Guess you learn new things every day... :wink:

**Pull request merged**